.circleci/config.yml

```diff
@@ -111,8 +111,8 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Install golangci-lint kubebuilder+env helm
-          command: make -j4 install-lint-deps install-kubebuilder install-helm install-controller-gen
+          name: Install binaries
+          command: make -j5 install-lint-deps install-kubebuilder install-helm install-controller-gen install-datadog-ci
```

Comment:
`j5` is it a type of instance?

Reply:
`-j` is a make flag that allows to run several make target in [parallel](https://www.gnu.org/software/make/manual/html_node/Parallel.html).

Here we install all program in parallel (low CPU, better to have high concurrency for network calls)